spec/acceptance/default_parameters_jira_10_spec.rb

```diff
@@ -5,7 +5,7 @@
 def on_supported_os
   family = fact('os.family')
   major = fact('os.release.major')
-  (family == 'Debian' and (major >= '22.04' or (major >= '11' and major <= '20'))) or (family == 'RedHat' and major >= '8')
+  family == 'Debian' and (major >= '22.04' or (major >= '11' and major <= '20'))
```

Comment:
I'd maybe explicitly specify Ubuntu and Debian distros here instead of family + major.. but up to you

Reply:
> I'd maybe explicitly specify Ubuntu and Debian distros here instead of family + major.. but up to you

Yes, I think that would be better. I'll make another PR for that.